index.html

```diff
@@ -1,11 +1,5 @@
 <html>
   <head>
-    <title>Pants Setup</title>
```

Comment:
It probably would be best to keep the LHS and just add the meta tag. If - for whatever reason - a refresh does not occur - having a title and and explanation of where to go is a good thing.